Makefile

```diff
@@ -114,7 +114,7 @@ manifests: controller-gen
 
 unit-test:
 	go clean -testcache
-	go test -p 8 -race $$(find pkg -type f -name '*_test.go' | sed -r 's|/[^/]+$$||' | sort | uniq | sed "s|^|volcano.sh/volcano/|")
+	go test -p 8 -race $$(find pkg cmd -type f -name '*_test.go' | sed -r 's|/[^/]+$$||' | sort | uniq | sed "s|^|volcano.sh/volcano/|")
```

Comment:
What's the background about the modifications?

Reply:
i see directory cmd have unit tests but not in use, so add it to unit-test in Makefile. i guess this is useful.